docs/basics/networkvariable.md

```diff
@@ -535,7 +556,7 @@ In order to create your own `NetworkVariableBase` derived container, you should:
     }
  ```
 
-While the above example is not the "recommended" way to synchronize a list that frequently changes (i.e. one or more elements position/order or add/remove), it is just an example of how you can "define your own rules" through using `NetworkVariableBase`.  Whether you handle managed or unmanaged value types is up to you.
+While the above example is not the "recommended" way to synchronize a list that frequently changes (i.e. one or more elements position/order or add/remove), it is just an example of how you can "define your own rules" through using `NetworkVariableBase`. 
```

Comment:
> While the above example is not the "recommended" way to synchronize a list that frequently changes (that is, one or more elements position/order or add/remove), it is just an example of how you can define your own rules using `NetworkVariableBase`. 

The "(that is, one or more elements position/order or add/remove)" part confuses me a little bit, though.